### PR TITLE
docs: README tfsec to trivy migration callout

### PR DESCRIPTION
**Pull request by AnaisUrlichs**
Related to the tasks in the following issue  #2016

### CODE REVIEW

**Comment by itaysk** [comment]
@AnaisUrlichs can you please update this and we can merge

**Comment by simar7** [comment]
Somewhat related to this PR: Should we have a change notice (IOW a preamble) as part of the tfsec welcome message (when tfsec is run) that also signals the above change? idea by @giorod3

**Comment by simar7** [comment]
I've cut a PR for the CLI message here https://github.com/aquasecurity/tfsec/pull/2067

Should we merge this PR? @AnaisUrlichs?

File: README.md [hunk 1]
```diff
@@ -10,7 +12,20 @@
 [![AUR version](https://img.shields.io/aur/version/tfsec-bin)](https://aur.archlinux.org/packages/tfsec-bin)
 [![VScode Extension](https://img.shields.io/visual-studio-marketplace/v/tfsec.tfsec?label=vscode)](https://marketplace.visualstudio.com/items?itemName=tfsec.tfsec)
 
-tfsec uses static analysis of your terraform code to spot potential misconfigurations.
+    
+> 📣 All of our scanning-related efforts have moved to one place, and that is Trivy. Please look at the [migration guide](https://aquasecurity.github.io/trivy/latest/tutorials/tfsec/migration/) and the Trivy [docs](https://aquasecurity.github.io/trivy/latest/) for further information.
```
Comment (mggude) [inline README.md]: This migration link is broken

Comment (mggude) [inline README.md]: This migration link is 404ing, should it be: https://github.com/aquasecurity/tfsec/blob/master/tfsec-to-trivy-migration-guide.md

**Comment by simar7** [comment]
ping @AnaisUrlichs?

**Review by simar7 — APPROVED** [review]

**Pull request merged**